index.html

```diff
@@ -7,10 +7,92 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" type="text/css" href="style.css">
 </head>
 
 <body>
-    <!-- Add your markup here -->
+<!-- Main Content -->
+<main>
+<header>
+<div class="header">
+    <div class="container">
+        <img class="logo" src="/images/logo.png" alt="logo">
+    </div>
+    <div class="container-right">
+        <h1>Your Cake Mate</h1>
+        <h2>For Your Orders <br> 07403720162</h2>
+    </div>
+</header>
+<nav>  
+  <div class="topnav">
+    <a href="#">Home</a>
+    <a href="#">Cakes</a>
+    <a href="#">Ordering</a>
+    <a href="#">Lessons</a>
+    <a href="#">About</a>
+  </div>
+</nav> 
+
+
+    <div class="row">
+        <div class="rightcolumn">
+            <div class="card">
+                <h2>CAROUSSEL</h2>
+                <div class="img" style="height:200px;">Image</div>
```

Comment:
Remember to put styling in the class rather than inline in the html element; this keeps all styling in one place. Should this also have a image element?